strax/processing/general.py

```diff
@@ -364,3 +366,14 @@ def _touching_windows(thing_start, thing_end,
         result[i] = left_i, right_i
 
     return result
+
+def _check_sorted(things, containers):
+    """
+    Check whether things and containers are sorted
+    :param things: Sorted array of interval-like data
+    :param containers: Sorted array of interval-like data
+    """
+    for array, name in zip([things, containers], ['things', 'containers']):
+        assert np.all(np.diff(array['time']) >= 0), f'{name} should be sorted'
```

Comment:
Is this truly a requirement do all time values to be non-zero? I do not think so.

Reply:
It is to say the differences between adjacent times are not negative.

Reply:
Yes, see  my comment below, I was being stupid.